test/base/src/androidTest/kotlin/io/realm/test/shared/QueryTests.kt

```diff
@@ -676,9 +807,131 @@ class QueryTests {
     }
 
     @Test
-    @Ignore
+    fun sum_find_emptyTable() {
+        val assertions = { propertyDescriptor: PropertyDescriptor ->
+            realm.query(QuerySample::class)
+                .sum(propertyDescriptor.property.name, propertyDescriptor.clazz)
+                .find { sumValue: Any? ->
```

Comment:
Something doesn't look right if this returns `Any?` Or is it because `propertyDescriptior.clazz` is a runtime variable?